libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -525,6 +525,25 @@ bool IRGeneratorForStatements::visit(BinaryOperation const& _binOp)
 	return false;
 }
 
+void IRGeneratorForStatements::generateSimpleFunctionCall(
+	FunctionCall const& _functionCall,
+	FunctionType::Kind _kind,
+	std::vector<ASTPointer<Expression const>> const& _arguments,
+	TypePointers const& _parameterTypes
```

Comment:
Do we really need all of these arguments? I think some of them are trivial to get from `_functionCall`

Reply:
I also don't see why we need a new function for this at all.